cmd/sriovdp/manager.go

```diff
@@ -100,6 +106,77 @@ func (rm *resourceManager) readConfig() error {
 	return nil
 }
 
+func (rm *resourceManager) storeConfig() error {
```

Comment:
storing the last config the daemon is using is also a nice debug feature IMO.
can you split this to a separate PR ? 

and should be optionally executed based on config from user via flag (e.g hostConfigDir was provided)